package.json

```diff
@@ -41,7 +41,7 @@
     "css-rtl": "cross-env NODE_ENV=RTL postcss --config build/postcss.config.mjs --dir \"dist/css\" --ext \".rtl.css\" \"dist/css/*.css\" \"!dist/css/*.min.css\" \"!dist/css/*.rtl.css\"",
     "css-lint": "npm-run-all --aggregate-output --continue-on-error --parallel css-lint-*",
     "css-lint-stylelint": "stylelint \"**/*.{css,scss}\" --cache --cache-location .cache/.stylelintcache",
-    "css-lint-vars": "fusv scss/ site/assets/scss/",
+    "css-lint-vars": "fusv scss/ site/assets/scss/ --ignoreFiles \"**/tokens/_raw.scss\"",
```

Comment:
Like you said, in the end it would be simpler to exclude the file, but for now we can comment unused variables since the full publication process is not ready. I don't have strong opinion regarding that.

Reply:
I would remove this then, as maybe we won't have anything to comment in the future if our Style Dictionary parser is smart enough.